src/wasm_worker.js

```diff
@@ -40,6 +40,7 @@ if (ENVIRONMENT_IS_NODE) {
   // Weak map of handle functions to their wrapper. Used to implement
   // addEventListener/removeEventListener.
   var wrappedHandlers = new WeakMap();
+  globalThis.onmessage = null;
```

Comment:
I could split the wasm workers stuff into a separate PR but I've already done a lot of splitting.  Let me know if you want me to split this one off too.